scripts/build-jammy-base-oracle.sh

```diff
@@ -38,16 +38,26 @@ fi
 ORACLE_CLOUD_NAME="$ORACLE_REGION-$ENVIRONMENT-oracle"
 [ -e "$LOCAL_PATH/../clouds/${ORACLE_CLOUD_NAME}.sh" ] && . $LOCAL_PATH/../clouds/${ORACLE_CLOUD_NAME}.sh
 
+[ -z "$IMAGE_ARCH" ] && IMAGE_ARCH="x86_64"
+
+if [[ "$IMAGE_ARCH" == "aarch64" ]]; then
+  SHAPE="$SHAPE_A_1"
```

Comment:
This won't allow overriding of SHAPE for aarch64.

Reply:
good call, checking shape first now